src/NzbDrone.Core/Movies/MovieRepository.cs

```diff
@@ -313,19 +313,20 @@ public List<int> AllMovieTmdbIds()
 
         public List<int> GetRecommendations()
         {
-            var recommendations = new List<List<int>>();
+            var recommendations = new List<int>();
             var tmdbIds = AllMovieTmdbIds();
 
             using (var conn = _database.OpenConnection())
             {
-                recommendations =  conn.Query<List<int>>("SELECT Recommendations FROM Movies ORDER BY id DESC LIMIT 100").ToList();
+                recommendations = conn.Query<int>(@"
+                     SELECT Recommendations FROM
+                    (SELECT Recommendations, count(*) AS recCount FROM
+                    (SELECT CAST(j.value AS INT) AS Recommendations FROM Movies CROSS JOIN json_each(Movies.recommendations) AS j
+                    WHERE Recommendations NOT IN (SELECT TmdbId FROM Movies))
+                    GROUP BY Recommendations ORDER BY recCount desc LIMIT 100);").ToList();
```

Comment:
SQL can be simplified, also the exisitng check doesn't work as you have the alias named the same as a column in movies table. Try this

```
SELECT Rec FROM
(SELECT CAST(j.value AS INT) AS Rec FROM Movies CROSS JOIN json_each(Movies.Recommendations) AS j
WHERE Rec NOT IN (SELECT TmdbId FROM Movies))
GROUP BY Rec ORDER BY count(*) DESC LIMIT 100;
```

Reply:
tested (in SQL browser only)  nice catch!  seems to work!